src/main/java/org/ow2/proactive/procci/model/occi/platform/Component.java

```diff
@@ -0,0 +1,94 @@
+package org.ow2.proactive.procci.model.occi.platform;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+import org.ow2.proactive.procci.model.exception.SyntaxException;
+import org.ow2.proactive.procci.model.occi.metamodel.Attribute;
+import org.ow2.proactive.procci.model.occi.metamodel.Kind;
+import org.ow2.proactive.procci.model.occi.metamodel.Link;
+import org.ow2.proactive.procci.model.occi.metamodel.Mixin;
+import org.ow2.proactive.procci.model.occi.metamodel.Resource;
+import org.ow2.proactive.procci.model.occi.platform.constants.PlatformAttributes;
+import org.ow2.proactive.procci.model.occi.platform.constants.PlatformKinds;
+import lombok.Getter;
+
+public class Component extends Resource {
+
+    @Getter
+    private Optional<Status> status;
+
+    public Component(Optional<String> url, Kind kind, Optional<String> title, List<Mixin> mixins,
+            Optional<String> summary, List<Link> links, Optional<Status> status) {
+        super(url, kind, title, mixins, summary, links);
```

Comment:
The same with Resource class: 6 arguments are too much. Usually 3 arguments are okay, 4 might be okay, 5 or more tell you to refactor.